pkg/kclient/operators.go

```diff
@@ -0,0 +1,16 @@
+package kclient
+
+import (
+	olmv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetClusterServiceVersionList returns a list of CSVs in the cluster
+// It is equivalent to doing `oc get csvs` using oc cli
+func (c *Client) GetClusterServiceVersionList() (*olmv1alpha1.ClusterServiceVersionList, error) {
+	csvs, err := c.OperatorClient.ClusterServiceVersions(c.Namespace).List(v1.ListOptions{})
```

Comment:
Are you asking to use `glog` to print a message when verbose mode is enabled?

Reply:
yes, info log

Reply:
I don't mind doing that but is there a criteria for when it should be added? Just that I don't know when it's a good idea to do that.

Reply:
no criteria, i just felt we could add a log line(debug/info) here, i have seen there are not enough logs in odo for debugging, it makes things easy when we cannot use breakpoints or in case error reported by user. Its your wish to add, it was a nit.

Reply:
There is no such thing as enough logs